tests/atest_mock.py

```diff
@@ -65,23 +66,38 @@ def executor(client):
         ],
         "getattrs 32802\r\n": [
             "303 simple attributes follow, terminated with '.'\r\n",
-            "state: working\r\nrouter: uninett-gsw2\r\ntype: bgp\r\nopened: 1524465142\r\nremote-addr: 2001:700:0:4515::5:11\r\nid: 32802\r\npeer-uptime: 0\r\nupdated: 1533116751\r\npolladdr: 128.39.103.25\r\npriority: 100\r\nbgpOS: down\r\nbgpAS: halted\r\nremote-AS: 64666\r\nlastevent: peer is admin turned off\r\n.\r\n",
+            "state: working\r\nrouter: uninett-gsw2\r\ntype: bgp\r\nopened: 1524465142"
+            "\r\nremote-addr: 2001:700:0:4515::5:11\r\nid: 32802\r\npeer-uptime: 0\r\n"
+            "updated: 1533116751\r\npolladdr: 128.39.103.25\r\npriority: 100\r\nbgpOS: "
+            "down\r\nbgpAS: halted\r\nremote-AS: 64666\r\nlastevent: peer is admin "
+            "turned off\r\n.\r\n",
```

Comment:
A line break here is a bad idea. Add `# noqa` or whatever it takes.

This is as raw a dump as is still readable straight from the line protocol, it acts as documentation. It might even be better to surround it with `"""` and then a `# noqa` . The point is it needs to be easily readable for a *human*. The `\r\n`s acts as a unit and must not be split up. The ugly formatting is the *point*.